src/client/renderer/r_array.c

```diff
@@ -350,10 +351,10 @@ void R_CreateInterleaveBuffer_(r_buffer_t *buffer, const GLubyte struct_size, co
 		Com_Warn("Buffer struct not aligned to 4, might be an error\n");
 	}
 
-	R_CreateBuffer(buffer, R_ATTRIB_TOTAL_TYPES, struct_size, false, hint, R_BUFFER_DATA | R_BUFFER_INTERLEAVE, size, data,
+	R_CreateBuffer(buffer, R_ATTRIB_TOTAL_TYPES, struct_size, false, false, hint, R_BUFFER_DATA | R_BUFFER_INTERLEAVE, size, data,
```

Comment:
When things get over 3-4 parameters, I think we should consider refactoring to take in a pointer to a struct, as we do for a lot of the cgame-visible entry points. That way, it's easy to keep track of the parameters, it's easy to pass `NULL` and `false` because you can just omit them, etc. So this could look something like:

    R_CreateBuffer(&(const r_create_buffer_t) {
        .id = id,
        .types = R_ATTRIB_TOTAL_TYPES,
        .size = size,
        .hint = hint,
        .flags = R_BUFFER_DATA | R_BUFFER_INTERLEAVE,
        .stride = struct_size,
        .data = data
    });

Thoughts?